src/ui/components/PrefixBadge.tsx

```diff
@@ -15,7 +15,6 @@ export default function PrefixBadgeButton({ breakpoint }: { breakpoint: Breakpoi
 
   return (
     <PrefixBadgePicker
-      key={breakpoint.options.prefixBadge}
```

Comment:
I'm not sure why we were doing this before, but it was breaking the prefix badge close animation. Now it works again.

Reply:
odd